src/jobflow/core/job.py

```diff
@@ -1331,6 +1332,7 @@ def prepare_replace(
         metadata = replace.metadata
         metadata.update(current_job.metadata)
         replace.metadata = metadata
+        replace._kwargs = current_job._kwargs
```

Comment:
I'm not sure about propagating the `_kwargs` for a single job replace.

For example, if you have the following setup, I would expect the data store of `job2` to be used, rather than job1.

```python
@job
def job1():
    replaced_job = job2()
    return Response(replace=replaced_job)

@job(data=["output"])
def job2():
    return {"output": my_big_data}
```

To give you a concrete example, if you have a job which updates the input set of a vasp calculation and then submits a replace, you would expect the data settings for the vasp job to be used rather than the initial job.

```python
def update_input_set(structure, vasp_maker):
    vasp_maker.input_set_generator.use_incar_settings["NSW"] = 100
    vasp_job = vasp_maker.make(structure)
    return Response(replace=vasp_job)
```

In this implementation, the data store settings of the `vasp_job` will be removed.

I think copying over the kwargs is only needed for the last `store_inputs` job of a replaced flow, since otherwise it is impossible to set it. For a single replaced job, you can just set the data store settings yourself.

Reply:
I see! Thanks. I will remove this and then it should hopefully be ready to merge!